app/main.py

```diff
@@ -16,8 +32,35 @@ def robots():
         return f.read()
 
 
+@app.post("/analyze_data")
+async def get_evaluation(board_data: EvaluationModel):
+    fen_position = board_data.fen
+    stockfish.set_fen_position(fen_position)
```

Comment:
Keep code simple. No need to expand things that can be written in one line, if it doesn't hurt readability or debugging:
```suggestion
    stockfish.set_fen_position(evaluation_request.fen)
```